src/main/osd/osd.h

```diff
@@ -259,9 +259,9 @@ typedef enum {
 // Make sure the number of warnings do not exceed the available 32bit storage
 STATIC_ASSERT(OSD_WARNING_COUNT <= 32, osdwarnings_overflow);
 
-#define ESC_RPM_ALARM_OFF -1
-#define ESC_TEMP_ALARM_OFF INT8_MIN
-#define ESC_CURRENT_ALARM_OFF -1
+#define ESC_RPM_ALARM_OFF 		-1
+#define ESC_TEMP_ALARM_OFF 		0
+#define ESC_CURRENT_ALARM_OFF 	-1
```

Comment:
A few tab stops snuck back in there. Sorry for being a pest 😃 On a positive note this all looks reeeally good imo.

Reply:
Fixed. Thanks for your time :pray: